spring-boot-project/spring-boot-tools/spring-boot-test-support/src/main/java/org/springframework/boot/testsupport/testcontainers/BitnamiImageNames.java

```diff
@@ -32,7 +32,7 @@ public final class BitnamiImageNames {
 
 	private static final String MARIADB_VERSION = "11.2.3";
 
-	private static final String MONGO_VERSION = "7.0.5";
```

Comment:
This doesn't look right.

Reply:
This was necessary to get the build green (see https://github.com/bitnami/containers/issues/61882)
The image seems to be fixed now, I will remove the commit.